CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -388,40 +403,38 @@ class CameraFragment : Fragment() {
                     // Display flash animation to indicate that photo was captured
                     container.postDelayed({
                         container.foreground = ColorDrawable(Color.WHITE)
-                        container.postDelayed(
-                                { container.foreground = null }, ANIMATION_FAST_MILLIS)
+                        container.postDelayed({container.foreground = null}, ANIMATION_FAST_MILLIS)
```

Comment:
I think `{` and `}` need to have spaces after and before, respecitvely, as per the Kotlin style guide